src/gcp/storage.js

```diff
@@ -56,16 +56,16 @@ async function _uploadObject(source, bucketName) {
   if (path.extname(source.file) !== ".zip") {
     throw new FirebaseError(`Expected a file name ending in .zip, got ${source.file}`);
   }
-  const location = `/v0/b/${bucketName}/o/${path.basename(source.file)}`;
-  await api.request("POST", location, {
+  const location = `/${bucketName}/${path.basename(source.file)}`;
+  await api.request("PUT", location, {
```

Comment:
Curious about this being a put - does the storage API upsert if nothing exists at that location?

Reply:
Yep, GCS recommends using `PUT` instead of `POST` (https://cloud.google.com/storage/docs/xml-api/post-object) when creating an object like this.

I think we can use `POST` by making calls to `https://storage.googleapis.com/upload/storage/v1/b/[BUCKET_NAME]/o?uploadType=media&name=[OBJECT_NAME]` instead (note the `upload` in the URL path), but PUT endpoint seems easier to hit.